packages/admin/cms-admin/src/userPermissions/user/permissions/ContentScopeGrid.tsx

```diff
@@ -56,57 +55,94 @@ export const ContentScopeGrid: React.FC<{
         },
     );
 
-    if (error) throw new Error(error.message);
+    const deleteContentScope = (row: GQLUserContentScopes[number]) => {
+        if (!data) return;
+        const modiefiedData = data?.userContentScopes.filter(
+            (contentScope: GQLUserContentScopes[number]) => JSON.stringify(contentScope) != JSON.stringify(row),
+        );
 
-    if (!data) {
-        return <Loading />;
+        submit(modiefiedData);
+    };
+    let columns: GridColDef<GQLUserContentScopes>[] = [];
+
+    if (!loading) {
+        columns = [
+            ...Object.keys(data?.userContentScopes[0]).map((key) => ({
+                field: key,
+                flex: 1,
+                pinnable: false,
+                headerName: camelCaseToHumanReadable(key),
+                renderCell: ({ row }: GQLUserContentScopes[number]) => camelCaseToHumanReadable(row[key]),
+            })),
+            {
+                field: "actions",
+                width: 120,
+                headerName: "",
+                sortable: false,
+                pinnable: false,
+                filterable: false,
+                renderCell: ({ row }) => (
+                    <>
+                        <IconButton
+                            color="inherit"
+                            onClick={() => {
+                                deleteContentScope(row);
+                            }}
+                        >
+                            <Delete />
+                        </IconButton>
+                    </>
+                ),
+            },
+        ];
     }
 
+    if (error) throw new Error(error.message);
+
+    const deleteSelected = () => {
+        if (!data) return;
+        const modiefiedData = data?.userContentScopes.filter((contentScope: GQLUserContentScopes[number]) =>
+            selectionModel.map((row) => JSON.stringify(row)).includes(JSON.stringify(contentScope)),
+        );
+
+        submit(modiefiedData);
+    };
```

Comment:
How can I trigger this action? Selecting rows in the data grid doesn't do anything locally...

Reply:
At the moment this function isn´t used and will be added together with the context menu

Reply:
Could you please rebase this on #2115?